Documentation/Developer/Index.rst

```diff
@@ -137,9 +137,9 @@ Each property has these fields:
 +-----------------+-----------------------------------------+----------------------------------------------------------------------------+
 
 
-The frontend form handling of the property types file and image are not yet implemented, due to a missing implementation in extbase.
-You have to implement the upload handling by yourself! The property select list has no configuration options in the modeler
-yet. You have to add the items list in the TCA configuration.
+The frontend form handling of the property types file and image are not yet implemented, due to a missing implementation in Extbase.
+The upload handling must be coded by the programmer! The property select list has no configuration options in the modeler
+yet. The items list must be inserted into the TCA configuration.
 
```

Comment:
> The property select list has no configuration options in the modeler yet. The items list must be inserted into the TCA configuration.

This is not clear for me - before and after the edit.

Reply:
The TCA can have a select box. The chosable items of the select box are coded in the TCA of this table.
However this documentation should not eplain these details.